### ISSUE TITLE
Download translation with Composer

### ISSUE THREAD

**Issue by titouanmathis** [studiometa/create-wordpress-project#1]
We should download FR translations via Composer in order to have a translated dashboard.

We can use one of this package:
- https://github.com/inpsyde/wp-translation-downloader
- https://github.com/wp-languages/wp-languages.github.io

A more thorough search/benchmark could be made to find the best solution!

**Comment by perruche** [comment]
I have tested `wp-translation-downloader` with great success. Did not test the other one as it was working cleanly

**Comment by ptt-homme** [comment]
I will have a look to the other one, but the solution seems legit! ;-)

**Comment by notjb** [comment]
How about using wp cli instead ?
👀  https://developer.wordpress.org/cli/commands/language/ 

This could be added to the readme.md file right after the step "Install WordPress with WPCLI" : 

```
**Optional**
# Install a core language pack.
`./vendor/bin/wp language core install fr_FR`
`./vendor/bin/wp language core activate fr_FR`
```

**Comment by notjb** [comment]
closed by #56
